shell/browser/ui/message_box_win.cc

```diff
@@ -148,10 +148,8 @@ int ShowTaskDialogUTF16(NativeWindow* parent,
 
   if (!checkbox_label.empty()) {
     config.pszVerificationText = checkbox_label.c_str();
-
-    if (checkbox_checked && *checkbox_checked) {
+    if (checkbox_checked && *checkbox_checked)
```

Comment:
Since `checkbox_checked` is a bool, this line can be just `if (checkbox_checked)`